changelog.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+### Added
+- Add: improved linting in CI, including `black`, `isort`, `flake8`, and `fawltydeps` (@obscurerichard, #496)
+
+### Contributors to this release
+
+@obscurerichard
```

Comment:
```suggestion
```

we will add contributors at the end in our changelog right before a release is crafted.